catalog.html

```diff
@@ -0,0 +1,262 @@
+              </p>
+              <p class="raiting">
+                Рейтинг: 5,0
+              </p>
+            </li>
+          </ul>
+          <ul class="pagination">
+            <li class="pagination-item">
+              <a class="pagination-link pagination-current">1</a>
+            </li>
+            <li class="pagination-item">
+              <a class="pagination-link" href="#">2</a>
+            </li>
+            <li class="pagination-item">
+              <a class="pagination-link" href="#">3</a>
+            </li>
+            <li class="pagination-item">
+              <a class="pagination-link" href="#">4</a>
+            </li>
+            <li class="pagination-item">
+              <a class="pagination-link pagination-link-next">...</a>
+            </li>
+            <li class="pagination-item">
+              <a class="pagination-link" href="#">10</a>
+            </li>
+          </ul>
+        </section>
+        <section class="newsletter">
+          <h2 class="newsletter-title-catalog">Подпишитесь на рассылку</h2>
+          <p class="newsletter-text-catalog">Только полезная информация и никакого спама, честное бойскаутское!</p>
+          <form class="newsletter-form" action="https://echo.htmlacademy.ru/" method="post">
+            <label class="visually-hidden" for="newsletter-email">Email</label>
+            <input class="field" placeholder="Ваш e-mail" type="email" name="newsletter-email" id="newsletter-email" required>
+            <button class="button newsletter-button" type="submit">Подписаться</button>
+          </form>
+        </section>
+    </main>
+    <footer class="page-footer">
+      <section class="social">
+        <h2 class="visually-hidden">Связаться с нами!</h2>
+        <ul class="social-list">
+          <li class="social-item">
+            <a class="social-link social-link-vk" href="https://vk.com/htmlacademy">
+              <span class="visually-hidden">Мы ВКонтакте</span>
+            </a>
+          </li>
+          <li class="social-item">
+            <a class="social-link social-link-tg" href="https://t.me/htmlacademy">
+              <span class="visually-hidden">Мы в телеграм</span>
+            </a>
+          </li>
+          <li class="social-item">
+            <a class="social-link social-link-youtube" href="https://www.youtube.com/user/htmlacademyru">
+              <span class="visually-hidden">Смотрите нас на Ютубе</span>
+            </a>
+          </li>
+        </ul>
+      </section>
+      <section class="contacts">
+        <h2 class="visually-hidden">Наши контакты</h2>
+        <a class="contacts-phone" href="tel:+78128121212">
+          +7 (812) 812-12-12
+        </a>
+      </section>
+      <section class="developer">
+        <h2 class="visually-hidden">Сайт разработчика</h2>
+        <a class="button-official-site" href="https://htmlacademy.ru/">
+          <img class="oficial-site" width="115" height="33" alt="Сайт разработан HTML-Academy">
```

Comment:
```suggestion
          <img class="official-img" width="115" height="33" alt="Сайт разработан HTML-Academy">
```

Такой блок принято называть copyright
И твоя любимая опечатка